tests/HasTagsTest.php

```diff
@@ -144,6 +144,20 @@ public function it_provides_as_scope_to_get_all_models_that_have_any_of_the_give
     }
 
     /** @test */
+    public function it_provides_as_scope_to_get_all_models_that_have_any_of_the_given_tag_instances()
+    {
+        $tag = Tag::findOrCreate('tagA');
```

Comment:
Could you use a typed tag in this test. imho that'll make it more clear that that instance of tag is used in `withAnyTags`.